### PR TITLE
Add exit with ctrl+c

### LINKED ISSUE #43: Cannot exit mitype with CTRL+C on windows
### Subject of the issue
The app is not quitting in test or replay mode using ctrl+c. It only quits in test mode pressing esc.

### Your environment
Windows 10
0.2.0
3.7.3
Windows cmd

### Steps to reproduce
1. Run the app
2. Press ctrl+c

Or 
1. Run the app
2. Finish a test
3. Press enter
4. Press ctrl+c

### Expected behaviour
The app should exit

### Actual behaviour
In test mode, "^C" appears in the lower area where the current word usually appears. On the other hand, nothing happens in replay mode. In fact, the app can only be closed by closing the cmd window from that point on.

### Your proposed fix (if any)
I believe the app is registering ctrl+c as a keystroke for the typing test and not as the usual "exit program" command. I would like to fix this for this year's Hacktoberfest.

### PR DESCRIPTION
**Pull request by diegoramosb**
This PR fixes #43 

Tested with python version - 3.7.3

On operating system - Windows 10

### CODE REVIEW

**Comment by Mithil467** [comment]
Please make sure the code is linted with black

**Comment by diegoramosb** [comment]
It should be good now. Sorry about that.

File: mitype/keycheck.py [hunk 1]
```diff
@@ -24,6 +24,36 @@ def is_escape(key):
...
+    This is used to exit the application.
+
+    Args:
+        key (string):
+
+    Returns:
+        bool: Returns true if Ctrl+c is pressed.
+            Returns false otherwise.
+    """
+    return key == "\x03"
```
Comment (Mithil467) [inline mitype/keycheck.py:54]: This function has been repeated

**Review by Mithil467 — CHANGES_REQUESTED** [review]

**Comment by Mithil467** [comment]
This fix works great! Just remove the duplicate function and we are good to merge this :)

**Review by Mithil467 — APPROVED** [review]
Thank you for the fix! @diegoramosb 😄

**Pull request merged**